.github/workflows/stale.yml

```diff
@@ -0,0 +1,49 @@
+name: Close inactive issues and PRs
+on:
+  schedule:
+    - cron: "30 1 * * *"
```

Comment:
Which timezone is this?

Reply:
I fear this is undefined when using GitHub runners.

There is an issue where others have requested the possibility of setting a timezone: https://github.com/actions/runner/issues/1423

If this is important, I could do further research.

Reply:
No, not important.